kura/org.eclipse.kura.driver.ble.xdk/OSGI-INF/metatype/org.eclipse.kura.driver.ble.xdk.xml

```diff
@@ -29,7 +29,7 @@
             cardinality="0"
             required="true"
             default="flase"
-            description="Enable Rotation-Quaternion, this chosen disability the Accelerometer and Gyroscope"/>
+            description="Enable Rotation-Quaternion and disability the Accelerometer and Gyroscope"/>
```

Comment:
I believe that "disability" is not the proper English term for this context. Please review the overall description to make it clear for the user